smithy-aws-traits/src/main/resources/META-INF/smithy/aws.protocols.smithy

```diff
@@ -123,7 +123,42 @@ structure ec2Query {}
 string ec2QueryName
 
 /// Indicates that an operation supports checksum validation.
-@trait(selector: "operation")
+@trait(
+    selector: "operation",
+    breakingChanges: [
+        {
+            change: "remove",
+            severity: "ERROR",
```

Comment:
Error is the default severity for breaking changes, so we can probably remove that here.

Reply:
I noticed that the request compression rules use `DANGER` [see](https://github.com/smithy-lang/smithy/blob/e3348620935216be2943f92c5288ec0a6762353e/smithy-model/src/main/resources/software/amazon/smithy/model/loader/prelude.smithy#L1253). These seem like they would have the same severity.

Reply:
I debated it and felt removal of compression being less severe than removal of checksums was reasonable. But don't feel strongly about it, so changing to DANGER.